features/admin.applications.v1/pages/applications-settings.tsx

```diff
@@ -0,0 +1,430 @@
+    props: ApplicationsSettingsPropsInterface
+): ReactElement => {
+    
+    const { 
+        mandateSSA,
+        authenticationRequired,
+        ssaJwks,
+        dcrEndpoint,
+        enableFapiEnforcement,
+        ["data-componentid"]: componentId 
+    } = props;
+   
+    const dispatch: Dispatch = useDispatch();
+    const { t } = useTranslation();
+    const { getLink } = useDocumentation();
+
+    const [ isAuthenticationRequired = true , setAuthenticationRequired ] = useState<boolean>(authenticationRequired);
+    const [ isMandateSSA = !isAuthenticationRequired, setMandateSSA ] = useState<boolean>(mandateSSA);
+    const [ isEnableFapiEnforcement, setEnableFapiEnforcement ] = useState<boolean>(enableFapiEnforcement);
+    const [ ssaJwksState, setSsaJwks ] = useState<string>(ssaJwks);
+    const [ dcrEndpointState, setDCREndpoint ] = useState<string>(dcrEndpoint);
+    const [ isSubmitting, setIsSubmitting ] = useState<boolean>(false);
+
+
+    const {
+        data: dcrConfigs,
+    } = useGetDCRConfigurations();
+
+    // set initial values
+    useEffect(() => {
+        if (dcrConfigs) {
+            setMandateSSA(dcrConfigs?.mandateSSA);
+            setAuthenticationRequired(dcrConfigs?.authenticationRequired);
+            setEnableFapiEnforcement(dcrConfigs?.enableFapiEnforcement);
+            setSsaJwks(dcrConfigs?.ssaJwks);
+        }
+    }, [ dcrConfigs ]);
+
+    useEffect(() => {
+        if (dcrEndpoint !== undefined) {
+            return;
+        }
+
+        // Fetch the server endpoints for OIDC applications.
+        getOIDCApplicationConfigurations()
+            .then((response) => {
+                setDCREndpoint(response.dynamicClientRegistrationEndpoint);
+            })
+            .catch(() => {
+                dispatch(addAlert({
+                    description: t("applications:notifications.fetchOIDCIDPConfigs.genericError.description"),
+                    level: AlertLevels.ERROR,
+                    message: t("applications:notifications.fetchOIDCIDPConfigs.genericError.message")
+                }));
+            });
+    });
+
+    const handleUpdateSuccess = () => {
+        dispatch(
+            addAlert({
+                description: t("jwtPrivateKeyConfiguration:notifications.success.description"),
+                level: AlertLevels.SUCCESS,
+                message: t(
+                    "console:develop.pages.applicationsSettings.notifications.success.message"
+                )
+            })
+        );
+    };
+
+    const resolveConnectorUpdateErrorMessage = (error: AxiosError): string => {
+        
+        return (
+            t("console:develop.pages.applicationsSettings.notifications.error.description",
+                { description: error.response.data.description })
+        );
+    };
+
+    const handleUpdateError = (error: AxiosError) => {
+        if (error.response && error.response.data && error.response.data.detail) {
```

Comment:
We can simplify as follows with null check
```suggestion
        if (error?.response?.data?.detail) {
```